src/node.cc

```diff
@@ -689,6 +689,68 @@ const char* signo_string(int signo) {
   }
 }
 
+// These are all flags available for use with NODE_OPTIONS.
+//
+// Disallowed flags:
+//  These flags cause Node to do things other than run scripts:
+//    --version / -v
+//    --eval / -e
+//    --print / -p
+//    --check / -c
+//    --interactive / -i
+//    --prof-process
+//    --v8-options
+//  These flags are disallowed because security:
+//    --preserve-symlinks
+const char* const environment_flags[] = {
```

Comment:
to fit with other constant naming conventions, should this be `kEnvironmentFlags`?

Reply:
That's a good question!  :smile:

Reply:
I don’t think we’ve use the `kSomething` notation for non-primitives in C++ so far. This can stay as-is, if you like (ditto below)